BEP7.md

```diff
@@ -0,0 +1,320 @@
+# BEP-7: Non-fungible Tokens on Binance Chain
+
+- [BEP-7: Non-fungible Tokens on Binance Chain](#bep-7-non-fungible-tokens-on-binance-chain)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+    - [Couple things to keep in mind](couple-things-to-keep-in-mind)
+  - [4. Motivation](#4-motivation)
+  - [5. Collection Specification](#5-collection-specification)
+    - [5.1 Collection Properties](#51-collection-properties)
+    - [5.2 Collection Management Operation](#52-collection-management-operation)
+      - [5.2.1 Issue collection](#521-issue-collection)
+      - [5.2.2 Increase total supply](#522-increase-total-supply)
+  - [6. Token Specification](#6-token-specification)
+    - [6.1 Token Properties](#61-token-properties)
+    - [6.2 Token Management Operation](#62-token-management-operation)
+      - [6.2.1 Issue token](#621-issue-token)
+      - [6.2.2 Transfer Tokens](#622-transfer-tokens)
+      - [6.2.3 Freeze Tokens](#623-freeze-tokens)
+      - [6.2.4 Unfreeze Tokens](#624-unfreeze-tokens)
+  - [7. Token Use Case Examples](#7-token-use-case-examples)
+  - [8. License](#8-license)
+  - [9. Reference](#9-reference)
+
+## 1. Summary
+
+This BEP describes a proposal for non-fungible tokens(NFTs) management on the Binance Chain.
+
+## 2. Abstract
+
+BEP-7 Proposal describes a common set of rules for non-fungible token within the Binance Chain ecosystem. It introduces the following details of a non-fungible token on Binance Chain:
+
+- What information makes a non-fungible token on Binance Chain
```

Comment:
type NFT struct {
          owner            sdk.ACC.Address
`json: "owner"`
         Name             String
`json: "name"`
         Description    String
`json: "image"`
         TokenURI .     String
`json: "token_URI"`
}
@vshvsh packet goes origin->Hub->destination making a footprint in three chains move a description to the external metadata